utils.py

```diff
@@ -0,0 +1,16 @@
+import numpy as np
+from torch.utils import data
+
+class SplittedDataLoader:
+    def __init__(self, dataset: data.Dataset, batch_size:int, split=(4, 1)):
+        dataset= dataset
+        val_q = len(dataset) // sum(split) * split[1]
+
+        np.random.seed(2020)
+        val_indices = np.random.choice(list(range(len(dataset))), val_q)
+        train_indices = list(set(range(len(dataset))).difference(val_indices))
+        train_sampler = data.SubsetRandomSampler(train_indices)
+        vaild_sampler = data.SubsetRandomSampler(val_indices)
+
+        self.train = data.DataLoader(dataset, sampler=train_sampler, batch_size=batch_size)
+        self.val = data.DataLoader(dataset, sampler=vaild_sampler, batch_size=batch_size)
```

Comment:
val_batch_size is commonly not the same as train - for train you typically want to use larger batch_size and val_batch_size is calculated as maximum that can be fitted in GPU memory